src/main/frontend/format/block.cljs

```diff
@@ -257,6 +257,26 @@
               body)]
     (assoc block :body body)))
 
+(defn extract-filetags
+  [content]
+
+  (def filetags-prefix "#+filetags: ")
```

Comment:
not sure if it's a conventional way (e.g. is it the right way to define nested constants/functions in clojure?)

Reply:
You can define local constants or functions using `let`, for example:
```
(defn a-function
  [args]
  (let [binding-a "value"]
     ... do something
     ))
```

Reply:
Ah, thanks! So `def/defn` would define in the global scope (even though it's 'inside' `extract-filetags`), right? I think I experimented in some dodgy online interpreter at first, so didn't spot the difference.

Reply:
Yes, `def/defn` will define in the global scope, it's easy to make the mistake if you play with scheme :)